x-pack/plugins/security_solution/common/endpoint/data_loaders/index_endpoint_hosts.ts

```diff
@@ -124,8 +124,9 @@ export async function indexEndpointHostDocs({
     generator.updateHostData();
     generator.updateHostPolicyData();
 
+    // timestamp - timeBetweenDocs * (numDocs - j - 1),
     hostMetadata = generator.generateHostMetadata(
-      timestamp - timeBetweenDocs * (numDocs - j - 1),
+      timestamp,
       EndpointDocGenerator.createDataStreamFromIndex(metadataIndex)
     );
```

Comment:
this seems like a strange change cc @paul-tavares

Reply:
I remember copying code similar to this from/to another place in the data loaders. I had to do it in order for the documents to get picked up by the transforms (I think @joeypoon also had to do something similar).  It had something to do with ensuring that we did not send a document whose timestamp had lapse the transform process windows

@joeypoon do you remember? I also know we use these data loaders from FTR and you were just making changes there recently to improve it.

Also - if there is a better way to do this, we should change it. Maybe Stop the transform, then restart it? 🤷‍♂️

Reply:
For some reason the script was loading alerts into the future? This fixed that.. Not sure if you've also seen that or if it was a design feature or me not using it properly...

Reply:
yeah, I remember seeing that @dhurley14 . But now I wonder if your change broke the loading of the Endpoints. 

@joeypoon , @pzl - can you checkout this PR and test it?  Load some endpoints (ensure you use the `--fleet` flag) and then validate that they all show up on the Endpoint List.

Reply:
I don't recall this specific piece of code but the `'@timestamp': hostMetadata['@timestamp'] + 60000` slightly below this I definitely added to get around the transform checkpoint times as you mentioned @paul-tavares. We do actually use the stop transform, insert, then start transform instead [in some places now](https://github.com/elastic/kibana/blob/0c43f86470bb4cc52969e434103e654a854c2c57/x-pack/test/security_solution_endpoint_api_int/apis/metadata.ts#L50) too as you mentioned. I agree that this is a better solution.

Will test this out locally.

Reply:
To my slight confusion, loading endpoints still works. I got 30/30 loaded. I'd need to dig a bit deeper to see why it's working without the offset now. We did change the transform delay back to `4s` recently from `1s` but IIRC, when this offset was initially added, we were already at a `4s` delay.

Reply:
Might be worth removing with this PR, even if it is a change needed for endpoint. That way we don't increase the surface level of possible bugs here.

Reply:
I believe that's correct.